packages/legend-application-studio/src/stores/editor-state/element-editor-state/ClassState.ts

```diff
@@ -234,6 +235,8 @@ export class ClassState {
   class: Class;
   derivedPropertyStates: DerivedPropertyState[] = [];
   constraintStates: ConstraintState[] = [];
+  propertyColumns: Property[] = [];
```

Comment:
In terms of organizing this, I think it might be better to add a class called `ClassPropertyState`. Similar to the other property states. This can hold some of the drag states you have added to the metamodels.

Reply:
I put the drag states from property metamodel to the classeditor like recommended on the other comment and that made it work, hope that's ok!